### PR TITLE
Rewards to employees

### PR DESCRIPTION
**Pull request by mdobosz3**
Add list available rewards to employees

### CODE REVIEW

File: spec/system/reward_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+RSpec.describe 'Reward check', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+  let!(:reward) { create(:reward) }
+
+  it 'crud Reward for employee' do
```
Comment (ciejj) [inline spec/system/reward_spec.rb:13]: it's not crud, since we are testing index and show only :)

File: spec/system/reward_spec.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    visit root_path
+    click_link 'Log In'
```
Comment (ciejj) [inline spec/system/reward_spec.rb:15]: You are using login helpers in other specs - maybe use them here as well for the sake of consistenncy?

**Review by ciejj — APPROVED** [review]
Well done - I have just a couple of cosmetic suggestions regarding specs

**Review by oskarlakner — APPROVED** [review]
🍻

File: spec/system/reward_spec.rb (hunk 2 above)
Comment (mdobosz3) [reply spec/system/reward_spec.rb]: I used the login helpers :)

File: spec/system/reward_spec.rb (hunk 1 above)
Comment (mdobosz3) [reply spec/system/reward_spec.rb]: I changed the name of this spec

**Pull request merged**